### ISSUE TITLE
Import .ftpconfig - Unknown file format

### ISSUE THREAD

**Issue by Olli** [h3imdall/ftp-remote-edit#293]
As the title says - importing the .ftpconfig I got a 'Unknown file format' error. 
The config file of "remote-ftp" did work with the "remote-ftp" plugin. 
I can provide more infomations if needed.

**Comment by wacki4** [comment]
Can you put importing file content here? You can change host/name/password for other.

**Comment by Olli** [comment]
I hope this helps
```
{
    "protocol": "ftp",
    "host": "ftp.server.com",
    "port": 21,
    "user": "myuser",
    "pass": "mypassoword",
    "promptForPass": false,
    "remote": "/",
    "local": "./",
    "secure": true,
    "secureOptions": null,
    "connTimeout": 10000,
    "pasvTimeout": 10000,
    "keepalive": 10000,
    "watch": [],
    "watchTimeout": 500
}
```

**Comment by wacki4** [comment]
Imported file don't have any big letters? is it .ftpconfig file? Or other name?

**Comment by Olli** [comment]
it's named .ftpconfig

**Comment by cschindl** [comment]
@wacki4 

https://github.com/h3imdall/ftp-remote-edit/blob/cc1015e3d964fcaa945ca0f90b2f45a0185f35c2/lib/helper/import.js#L27

Problem is that "Path.extname(filePath)" returns empty string if file begins with a dot. (https://nodejs.org/api/path.html#path_path_extname_path)

**Comment by wacki4** [comment]
@cschindl so why did You change code which i made in such way ? 😄

**Comment by cschindl** [comment]
@wacki4 The error was already present in your code ;) I'll fix it right away.

**Comment by wacki4** [comment]
`} else if (Path.basename(filePath).toLowerCase() == '.ftpconfig') {`

I've only checked basename fo ftpconfig ;)

**Comment by cschindl** [comment]
@wacki4 ahhh you're right. My mistake.  Once too much copy &paste. ;)

**Comment by cschindl** [comment]
Fixed in release [0.16.1](https://github.com/h3imdall/ftp-remote-edit/releases/tag/v0.16.1). Please check if everything works correctly now.
